sphinx/source/docs/user_guide/server.rst

```diff
@@ -642,6 +642,36 @@ and the server, there is network traffic between the python client and the
 server as well. Depending on the particular usage, this could be a
 significant consideration.
 
+
+Embedding Bokeh Server as a Library
+-----------------------------------
+
+It can be useful to embed the Bokeh Server in a larger Tornado-based
+application with other concurrent network services or coroutines, while
+letting the application handle the lifetime of the I/O loop.  Here is how
+to integrate Bokeh in such a scenario:
+
+.. code-block:: python
+
+   def add_bokeh_server(loop, bokeh_applications, **server_kwargs):
+       """
+       Start a Bokeh Server instance in the Tornado *loop*
+       with the given *bokeh_applications*.
+
+       *server_kwargs* can contain additional Bokeh Server arguments,
+       such as the ``port`` number to listen on.
+       """
+       from bokeh.server.server import Server
+       server = Server(bokeh_applications, io_loop=loop,
+                       **server_kwargs)
+       # This returns immediately after having started all timers
+       # and services.
+       server.start()
```

Comment:
Perhaps consider just providing the body of the function here?  The function definition may not provide much here.